hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/router/ConnectionContext.java

```diff
@@ -150,8 +150,8 @@ public synchronized void close(boolean force) {
       // this is an erroneous case, but we have to close the connection
       // anyway since there will be connection leak if we don't do so
       // the connection has been moved out of the pool
-      LOG.error("Active connection with {} handlers will be closed",
-          this.numThreads);
+      LOG.error("Active connection with {} handlers will be closed, ConnectionContext is {}",
+          this.numThreads, this.toString());
```

Comment:
With LOG.error, I think you can just:
```
LOG.error("Active connection with {} handlers will be closed, ConnectionContext is {}",
          this.numThreads, this);
```

Reply:
Can you also show an example of what this would look like?

Reply:
> With LOG.error, I think you can just:
> 
> ```
> LOG.error("Active connection with {} handlers will be closed, ConnectionContext is {}",
>           this.numThreads, this);
> ```

Yes, fixed it. Thanks for this suggestion.

Reply:
> Can you also show an example of what this would look like?

Sir, I did not find this error log in our cluster. I just think we should log connection information here when some error occurs.